RecoTracker/MeasurementDet/BuildFile.xml

```diff
@@ -22,4 +22,3 @@
 <use name="RecoLocalTracker/Phase2TrackerRecHits"/>
 <use name="TrackingTools/DetLayers"/>
 <use name="TrackingTools/MeasurementDet"/>
-<use name="RecoLocalTracker/SiPhase2VectorHitBuilder"/>
```

Comment:
the following files are using this package in the /plugins:
```
MeasurementTrackerESProducer.cc:#include "RecoLocalTracker/SiPhase2VectorHitBuilder/interface/VectorHitBuilderAlgorithm.h"
TkStackMeasurementDet.cc:#include "RecoLocalTracker/SiPhase2VectorHitBuilder/interface/VectorHitBuilderAlgorithm.h"
TkStackMeasurementDet.h:#include "RecoLocalTracker/SiPhase2VectorHitBuilder/interface/VectorHitBuilderAlgorithm.h"
```
however there is no explicit or implicit dependence on it in the  RecoTracker/MeasurementDet/plugins/BuildFile.xml

IIUC, header-only dependencies are not required in BuildFile.
Incidentally, here, the file include is apparently not needed at all.

Please update the files listed above to drop the `#include "RecoLocalTracker/SiPhase2VectorHitBuilder/interface/VectorHitBuilderAlgorithm.h"` so that we have a more consistent and cleaner build file setup